examples/advanced_circuits_algorithms/Deutsch_Jozsa_Algorithm/Deutsch Jozsa Algorithm.ipynb

```diff
@@ -0,0 +1,1047 @@
+{
```

Comment:
<p>I like the first paragraph. I believe 2^n-1 should just be 2^n. Also, I think it might be good to change 'but this time on a quantum circuit.' to 'on a quantum circuit.'</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/177/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
Updated on most recent PR